crawl-ref/source/areas.cc

```diff
@@ -91,6 +111,11 @@ void areas_actor_moved(const actor* act, const coord_def& oldpos)
     }
 }
 
+/// \brief Add some of the actor's area effects to the grid and center caches
+/// \param actor The actor
+/// \details Adds an actor's silence, halo, liquidation, and umbra effect to
+/// the area grid (\ref _agrid) and center (\ref _agrid_centres) caches.
```

Comment:
In a few places (in this & other commits), you go into this level of detail, listing every current possible value for something. That's fragile, since if we add or remove area effects, we're quite unlikely to update this comment. Better to just talk about the category with maybe one example - "area effects (e.g. halos)" - and leave it at that.